frontend/packages/noobaa-storage-plugin/src/components/bucket-class/create-bc.scss

```diff
@@ -0,0 +1,91 @@
+/* top info styles */
+.nb-create-bc-info {
```

Comment:
Should We be using inline notifications for these?
<img width="885" alt="Screen Shot 2019-10-29 at 3 00 05 PM" src="https://user-images.githubusercontent.com/287880/67795319-d476c700-fa5c-11e9-9274-9686228d4735.png">

Reply:
I am not sure about it.. @shirimordechay should we be using this?

Reply:
@matthewcarleton @bipuladh I'm not sure what is usually used for contextual help, I worked according to this suggested design: http://openshift.github.io/openshift-origin-design/web-console/future-openshift/contextual-help/contextual-help

If contextual help is used in the form of the suggested inline notification somewhere else in the system, then yes, we should keep it consistent.

Reply:
ya I'm not sure the history on those designs. In Patternfly this inline notification was introduced I think for these types of messages. The specs in that design don't align with PatternFly 4 either so I'm concerned about that as well. It could be worth reaching out to the OpenShift side and see if anyone has opinions on it too. @beanh66

Reply:
sure

Reply:
The Console has `.co-hint-block` with blue styling that looks more like what @shirimordechay linked to. We may want to either:

A) Use `.co-hint-block` for now and discuss switching to PatternFly's inline alert component sometime after 4.3 (I have a feeling that Sam will want to make this switch everywhere all at once).

B) Not use hint blocks in this wizard and place the description text and learn more links (maybe without the "What is X?") directly into the top of each step instead. Some forms within the Console seem to have a sentence or two describing what the user is looking at, so not using hint blocks might align closer with that. I know this isn't a perfect example since it's not within a wizard, but here's the Create Secret form with some descriptive text at the top:

![image](https://user-images.githubusercontent.com/9122899/67900640-56d5b880-fb3b-11e9-8ad0-a1eb2a99b071.png)

Reply:
@bipuladh Can you create a another PR to fix the CSS issue. I think we can take this PR In as other PR's are dependent on this one. Also, paste the link for new PR here.

@andybraren @matthewcarleton @shirimordechay ^^